src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobManager.cs

```diff
@@ -873,7 +872,7 @@ async Task DeleteCheckpointDirectory(IEnumerable<string> blobsToDelete)
 
         #region ILogCommitManager
 
-        void ILogCommitManager.Commit(long beginAddress, long untilAddress, byte[] commitMetadata, long commitNum)
+        void ILogCommitManager.Commit(long beginAddress, long untilAddress, byte[] commitMetadata, long commitNum, bool forceWriteMetadata)
```

Comment:
How should we use `forceWriteMetadata` in this method?

Reply:
In case it helps, this seems defined here: https://github.com/microsoft/FASTER/blob/0116754bcf0857e614e2b7ea042b8db563cb26cc/cs/src/core/FasterLog/ILogCommitManager.cs#L23

and more context may be found here: https://github.com/microsoft/FASTER/blob/0116754bcf0857e614e2b7ea042b8db563cb26cc/cs/src/core/FasterLog/FasterLog.cs#L2077-L2083

Now that still doesn't make the answer obvious to me, but it gives me some clues. Perhaps this is something we can ask the FASTER folks about

Reply:
After looking at where this was  introduced in FASTER (https://github.com/microsoft/FASTER/pull/856) it is pretty clear that this fixes a problem that was introduced with "fast commit" where the commit is not always fully committed.

In our implementation we *always fully commit* anyway so we can ignore this argument.